027-FHIRPoweredHealthcare/readme.md

```diff
@@ -1,78 +1,69 @@
+19. **[IoT Central](https://docs.microsoft.com/en-us/azure/iot-central/healthcare/concept-continuous-patient-monitoring-architecture)** helps create, customize, and manage healthcare IoT solutions using IoT Central application templates. Continuous patient monitoring is one application template in healthcare IoT space.
 
 ## Scenario
-Contoso Healthcare is implementing a FHIR-based data management solution to rapidly exchange data in the HL7 FHIR standard format with EHR (Electronic Health Record) systems and HLS (Life Science) research databases.  To help its healthcare practitioners and administrators manage and access patient data for day-to-day operations, your team's assistance is needed in implementing new FHIR powered **[Health Architectures](https://github.com/microsoft/health-architectures)**.  This FHIR powered event-driven serverless platform will provide services to ingest and convert patient data from EMR (Electronic Medical Record), Clinical Data, Lab System, Scheduling System, etc. into FHIR Bundles and persist them into a FHIR Compliant store in near real-time.
-
-Your team's assistance is needed to implement this new event-driven FHIR ecosystem to build-out the following scenarios:
-1. Ingest and process patient record in HL7 FHIR or legacy formats from EHR systems into a common FHIR-based standard format and persist them into a FHIR Compliant store.
-2. Generate FHIR CUD (create, update, or delete) events whenever FHIR CUD operations take place in FHIR Server for post-processing.
-3. Securely connect and read FHIR patient data from FHIR Server through a web app and add a patient lookup feature to improve user experience.
-4. Explore a patient's medical records and encounters in FHIR Patient Dashboard and SMART on FHIR apps.
+Contoso Healthcare is implementing a FHIR-based data management solution. 
+Your team's assistance is needed to implement the following scenarios using FHIR Powered Healthcare hack:
+   * Ingest and process patient record in HL7 FHIR or legacy formats from EHR systems into a common FHIR-based standard format and persist them into a FHIR Compliant store.
+   * Generate FHIR CUD (create, update, or delete) events whenever FHIR CUD operations take place in FHIR Server for post-processing.
+   * Securely connect and read FHIR patient data from FHIR Server through a web app and add a patient lookup feature to improve user experience.
+   * Explore a patient's medical records and encounters in FHIR Patient Dashboard and SMART on FHIR apps.
+   * Bulk export data from FHIR, de-identify the data and store in Data Lake for further processing.
+   * Ingest and Persist IoT device data from IoT Central using IoT Connector for FHIR.
+   * Analyze and Visualize data (EHR, HL7, CCDA, IoMT) in PowerBI using PowerQuery Connector for FHIR and SQL DB Connector.
 
 ## Challenges
+<center><img src="./images/challenges_architecture.jpg" width="850"></center>
+
+**Required challenges complete in order:**
```

Comment:
Wording change to make it more clear and consistent: 
"These challenges must be completed in order"
And the next one:
"These challenges can be completed in any order"